intro-HPC/ch_scoop.tex

```diff
@@ -0,0 +1,90 @@
+\chapter{SCOOP}
+\label{ch:scoop}
+
+SCOOP (Scalable COncurrent Operations in Python) is a distributed task module allowing
+concurrent parallel programming on various environments, from heterogeneous grids to supercomputers.
+
+It can used for projects that require lots of (small) tasks to be executed.
```

Comment:
maybe mention that this is an alternative to `worker` (and refer to the section on `worker` that we already have)